backend/dataall/modules/mlstudio/aws/sagemaker_studio_client.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+from dataall.aws.handlers.sts import SessionHelper
+from dataall.modules.mlstudio.db.models import SagemakerStudioUser
+from botocore.exceptions import ClientError
+
+logger = logging.getLogger(__name__)
+
+
+@staticmethod
```

Comment:
This should be wrapped in to class if we want to group the methods or we should delete `staticmethod`